docs/tutorials/motion/plan-motion-with-arm-gripper.md

```diff
@@ -0,0 +1,431 @@
+{{% tab name="Go" %}}
+Because of experimental API changes that are ongoing, we must pass in a `slam` service resource name when using the Motion service to Move.
+Add `"go.viam.com/rdk/services/slam"` to your import list to provide access to the `slam` service.
+This tutorial will not cover any other SLAM content.
+
+```go {class="line-numbers linkable-line-numbers"}
+// Generate a sample "start" pose to demonstrate motion
+testStartPose := spatialmath.NewPose(
+  r3.Vector{X: 510.0, Y: 0.0, Z: 526.0},
+  &spatialmath.OrientationVectorDegrees{OX: 0.7071, OY: 0.0, OZ: -0.7071, Theta: 0.0},
+)
+testStartPoseInFrame := referenceframe.NewPoseInFrame(referenceframe.World, testStartPose)
+_, err = motionService.Move(context.Background(), myArmResource, testStartPoseInFrame, worldState, nil, slam.Named(""), nil)
+if err != nil {
+  logger.Fatal(err)
+}
+```
+{{% /tab %}}
+{{< /tabs >}}
+
+<!-- TODO : In the future, we should add some specific information on the importance of the frame chosen as the point of reference for PoseInFrame variables -->
+<!-- ## Managing Pose References -->
+
+## Commanding Other Components to Move with the Motion Service
+
+This section will require you to add a new component to your robot.
+One device that is very commonly attached to the end of a robot arm is a **gripper**.
+Most robot arms pick up and manipulate objects in the world through the use of a gripper, so learning how to directly command a gripper to move is highly valuable.
+Though future Motion service commands will command the gripper to move, ultimately the arm is doing all of the work in situation.
+This is made possible by the fact that the Motion service considers other parts of the robot (through the **Frame System**) when calculating how to achieve the desired motion.
+
+### Add a Gripper Component
+
+We need to do several things to prepare a new Gripper component for motion.
+
+1. Go back to your robot configuration in App.
+2. Under the **COMPONENTS** section, add a new `gripper` component to your robot with the following attributes:
+
+* Choose `myGripper` as the name of this new component
+* Choose `gripper` as the desired **Type** selection
+* Choose `fake` as the desired **Model** selection
+
+3. Add a **Frame** to this component
+
+* Mark the parent as `myArm`
+* Mark the translation as something small in the +Z direction, ex. 90 mm
+* Leave the orientation as the default
+* Specify a "Box" for the geometry
+* Put in desired values for the Box's **Length**, **Width**, and **Height**, and the Box origin's **X**, **Y**, and **Z** values.
+
+4. Mark the `myArm` component as what the new `myGripper` *Depends On*
+5. Save this new robot configuration
+  * Your `viam-server` instance should update automatically.
+
+<!-- TODO : Add a picture of a sample gripper configuration -->
+
+Because the new gripper component is "attached" (via the parent specification in the Frame ) to `myArm`, we can produce motion plans using `myGripper` instead of `myArm` 
```

Comment:
```suggestion
Because the new gripper component is "attached" (with the parent specification in the Frame) to `myArm`, we can produce motion plans using `myGripper` instead of `myArm` 
```